packages/react-form/src/hooks/list/hooks/reducer.ts

```diff
@@ -158,6 +174,29 @@ function reduceList<Item extends object>(
         list: action.payload.list.map(initialListItemState),
       };
     }
+    case 'moveFieldItem': {
+      const newList = [...state.list];
+
+      if (
```

Comment:
It would be better throw an exception instead of silently ignoring bad values.

Reply:
👍